packages/graphql-docs-generator/src/generator/getFields.ts

```diff
@@ -77,9 +77,16 @@ function isGraphQLAggregateField(field) {
   if (
     field &&
     field.name == 'aggregateItems' &&
-    field.type?.ofType?.name == 'SearchableAggregateResult'
+    getBaseType(field.type) == 'SearchableAggregateResult'
   ) {
     return true;
   }
   return false;
+}
+
+function getBaseType(field) {
```

Comment:
can we please rename the argument to `type` instead of `field` as we are actually passing on the type of the field.

Reply:
Updated. The existing unit test case takes care of this scenario. 
https://github.com/aws-amplify/amplify-codegen/blob/master/packages/graphql-docs-generator/__tests__/generator/getFields.test.ts#L189-L257

The change here is to treat `[SearchableAggregateResult]`, `[SearchableAggregateResult]!` and `[SearchableAggregateResult!]!` in the same way.